.github/labeler.yml

```diff
@@ -0,0 +1,7 @@
+# PRs that only touch the docs folder
+documentation:
+- all: [docs/**/*]
+
+# PRs that only touch type files
+typescript:
+- all: [types/**/*]
```

Comment:
Can you add the `test/types/*` as well?

Reply:
@RafaelGSS - `test/types/*` is included in  `**/types/**/*` now.

> Did you tried to wrap them with "?

Yes, that was the problem.